### ISSUE TITLE
Unable to generate Client on Windows

### ISSUE THREAD

**Issue by madebysid** [prisma/prisma#9304]
### Bug description

I installed Windows 10 on a VirtualBox VM, and I'm unable to generate Client.

I see this error:

```
Error: Unable to require(`C:\Users\me\Desktop\prisma\node_modules\prisma\query_engine-windows.dll.node`)
 \\?\C:\Users\me\Desktop\prisma\node_modules\prisma\query_engine-windows.dll.node is not a valid Win32 application.
\\?\C:\Users\me\Desktop\prisma\node_modules\prisma\query_engine-windows.dll.node
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Running `prisma generate` via `cmd` and via GitBash throws the same error. The VM is very new, the only things I installed were Node and GitBash.

### How to reproduce

1. `yarn init --yes`
2. `yarn add prisma @prisma/client`
3. yarn prisma init
4. Change the Prisma schema to:
```
datasource db {
  provider = "sqlite"
  url      = env("DB_URL")
}

generator client {
  provider = "prisma-client-js"
}

model User {
        id String @id
        name String @default("DefaultName")
}
```
5. Create a SQLite file
6. `yarn prisma generate`

### Expected behavior

`yarn prisma generate` throws no errors.

### Prisma information

Added it in the reproduction steps

### Environment & setup

- OS: Windows 10 (installed on VirtualBox on macOS)
- Database: SQLite (although I don't think this matters)
- Node.js version: v14.17.6

### Prisma Version

I also cannot run `prisma -v`, I see this error:
```
Error: Command failed with exit code 3221225595: C:\Users\me\Desktop\prisma\node_modules\@prisma\engines\introspection-engine-windows.exe --version
error Command failed with exit code 1.

But my Prisma version is 3.0.2

**Comment by madebysid** [comment]
I tried installing Visual C++ as suggested by https://github.com/prisma/prisma/issues/6508#issuecomment-919483693 (you can do this by checking the box thatsays `Automaically install necessary tools` during Node.js installation) and that seems to fix this issue.

Not entirely sure why, but leaving this comment for reference and closing this!

**Comment by madebysid** [comment]
List of everything I did:

1. Create a new VirtualBox VM
2. Install Windows 10
3. Install NodeJS, do not check the `Automatically install necessary tools` during installation
4. Install GitBash (this is likely not relevant)
5. Install `yarn` (also not relevant probably)
6. In GitBash:
```
cd ~/Desktop
mkdir prisma
cd prisma
yarn init --yes
yarn add prisma @prisma/client
yarn prisma init
```

then change the `prisma/schema.prisma`, and add a model to it. Use the schema I used in the original post, if lazy

7. In GitBash: `yarn prisma generate`
